cif_core.dic

```diff
@@ -15345,6 +15350,25 @@ save_audit_contact_author.id
     _type.source                  Related
     _type.container               Single
     _type.contents                Word
+    _method.purpose               Definition
+    _method.expression
+;
+    with aca as audit_contact_author
+    count = 0
+    target = 0
```

Comment:
```suggestion
    target = ''
```

Reply:
An empty string might be better at conveying that the value is not required to be an integer.